app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -14,6 +14,10 @@ def create
       authenticate
     end
 
+    def enable_cookies
+      @shop = sanitize_shop_param(params)
```

Comment:
You could just reuse (call) `sanitized_shop_name` from `LoginProtection`. It does almost exactly and I like to avoid using `@shop` variable name as a name for shop's domain.